raft/raft.go

```diff
@@ -160,6 +160,9 @@ type Config struct {
 	// overflowing that sending buffer. TODO (xiangli): feedback to application to
 	// limit the proposal rate?
 	MaxInflightMsgs int
+	// MaxInflightBytes limits the number of in-flight bytes in append messages.
+	// Complements MaxInflightMsgs. Ignored if zero.
```

Comment:
> This effectively bounds the bandwidth-delay product. Note that especially in high-latency deployments setting this too low can lead to a dramatic reduction in throughput. For example, with a peer that has a round-trip latency of 100ms to the leader and this setting is set to 1mb, there is a throughput limit of 10mb/s for this group. With a RTT of 400ms, this drops to 2.5mb/s.

Reply:
Done. Is this what you refer as [Little's law](https://en.wikipedia.org/wiki/Little%27s_law)?

Reply:
I think so, where the "system" is "being inflight". "Customers" arrive (and leave) at a rate of X mb/s, they spend `latency` time in the system, and the product of those two is the "number of customers in the store" (inflight bytes).

Reply:
Added a note on the law too.